packages/authentication/README.md

```diff
@@ -53,6 +53,13 @@ For a tutorial on how to add **JWT** authentication to an application, see
 For some background on our design decisions, please read
 [Multiple Authentication strategies](./docs/authentication-system.md).
 
+## Note
+
+From `@loobpack/authentication@3.0.0`, type `UserProfile` is extracted to
```

Comment:
`From` => `Starting from`

Reply:
`UserProfile` needs to be imported from `@loopback/security` and it's not backward compatible with the one exported from `@loobpack/authentication@2.x`.